### DISCUSSION TITLE
How does isValid work really?

### DISCUSSION THREAD

**Discussion by heggemsnes** [houseform/houseform#46] [Q&A] [answered]
Hi!

Testing out this library and it seems quite nice. 

I am a bit confused with the "isValid" boolean, I would think this would be false if there are any errors, but it seems it can be true. See an example using Next.js here: https://stackblitz.com/edit/nextjs-t6js87?file=pages/index.js

Basically just try to submit the form and "isValid" is still true.

In order to get the UX of a disabled button that is enabled when validated correctly I'm doing this now:

```typescript
<button
  type="submit"
  disabled={!isValid || errors.length > 0 || !isDirty}
  className="btn btn-icon btn-dark-blue disabled:bg-opacity-50"
>
  Send
</button>
```
Any pointers here would be appreciated!

**Comment by crutchcorn** [comment, marked as answer]
This appears to be a regression in 1.3.0 - I apologize, I thought I had fixed this.

Please use `1.2.2` in the meantime - I'll work on fixing this ASAP

https://github.com/crutchcorn/houseform/issues/47

**Comment by crutchcorn** [comment]
Heads up - this issue has now been fixed in `1.3.1`. Thank you for your report
